app/models/dashboard_stat.rb

```diff
@@ -0,0 +1,22 @@
+class DashboardStat
+  attr_reader :manager, :manager_events
+
+  def initialize(manager_id)
+    @manager = User.find(manager_id)
+    @manager_events = @manager.events
+  end
+
+  def events
```

Comment:
The method name does not clearly state what it does. It name suggests that it is returning events, not size